ca.parallel_launcher.ParallelLauncher.json

```diff
@@ -0,0 +1,112 @@
+{
+	"id": "ca.parallel_launcher.ParallelLauncher",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.15-21.08",
+	"sdk": "org.kde.Sdk",
+	"command": "parallel-launcher",
+	"finish-args" : [
+		"--socket=x11",
+		"--socket=wayland",
+		"--socket=pulseaudio",
+		"--share=ipc",
+		"--share=network",
+		"--device=all",
+		"--filesystem=host",
+		"--allow=multiarch",
```

Comment:
why do you need this?

Reply:
RetroArch uses it, and I launch RetroArch

Reply:
it still doesn't explain why.

Reply:
Hmm, since I only ever run the N64 emulator cores, which are 64-bit, I may be able to remove this permission.

Reply:
Yep, it isn't required for my use case. Removed it.